### ISSUE TITLE
[terra-infinite-list] Missing aria-label prop for infinite list 

### ISSUE THREAD

**Issue by adrian-94** [cerner/terra-framework#997]
# Bug Report

## Description

Currently, there isn't any prop available to set the aria-label for the Infinite list. This causes an accessibility violation when the Infinite List is used as `listbox`. 
## Steps to Reproduce

1. Create an Infinite List with role set to **lisbox**.
2. Analyzing the component through axe plugin on chrome should throw an accessibility violation
`Ensures every ARIA input field has an accessible name `

## Expected Behavior

There should be no accessibility violations.
## Possible Solution

Expose ariaLabel prop to set the aria-label, similar to other Terra components.
## Environment

* Component Name and Version: terra-infinite-list  version-3.0.0
* Browser Name and Version: Chrome version-79.0.3945.88
* Node/npm Version: 8.9.2
* Webpack Version: 4.41.2
* Operating System and version (desktop or mobile):

## @ Mentions

@dkasper-was-taken
